### PR TITLE
docs: message ordering docs

### PR DESCRIPTION
**Pull request by Kononnable**

### CODE REVIEW

File: docs/data_router.md [hunk 1]
```diff
@@ -29,13 +29,15 @@ Below are the example data required by data router:
 {
     "schemaId": <UUID>,
     "objectId": <UUID>,
+    orderGroupId: <Optional<UUID>>,
```
Comment (frondeus) [inline docs/data_router.md:32]:
```suggestion
    "orderGroupId": <Optional<UUID>>,
```

File: docs/features/message_ordering.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- [Message ordering in CDL](#message_ordering_in_cdl)
+- [How to use it](#how_to_use)
+    - [Overall](#overall)
+    - [Communication through Apache Kafka](#kafka)
+    - [Communication through RabbitMQ](#rabbit)
+    - [Communication through RPC](#rpc)
+  
+
+## <a name="what_is"></a>What is message ordering 
+Message ordering is a guarantee that certain messages will be processed by CDL in the same order there were sent to the system. Proper ordering meter mostly to user applications which base their business logic on real time data where no approximation is allowed.
```
Comment (frondeus) [inline docs/features/message_ordering.md:18]:
```suggestion
Message ordering is a guarantee that certain messages will be processed by CDL in the same order there were sent to the system. Proper ordering matter mostly to user applications which base their business logic on real-time data where no approximation is allowed. 
```

**Review by frondeus — DISMISSED** [review]

File: docs/features/message_ordering.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    - [Pros and cons](#pros)
+- [Message ordering in CDL](#message_ordering_in_cdl)
+- [How to use it](#how_to_use)
+    - [Overall](#overall)
+    - [Communication through Apache Kafka](#kafka)
+    - [Communication through RabbitMQ](#rabbit)
+    - [Communication through RPC](#rpc)
+  
+
+## <a name="what_is"></a>What is message ordering
```
Comment (luke-biel) [inline docs/features/message_ordering.md:17]: Shouldn't the `a`'s be around text?

**Review by luke-biel — APPROVED** [review]

File: docs/features/message_ordering.md (hunk 3 above)
Comment (Kononnable) [reply docs/features/message_ordering.md:17]: That's not necessary - it's just an named anchor to part of the document(it's not interactive, it's just a pointer to a place).

**Pull request merged**